src/ReTestItems.jl

```diff
@@ -487,6 +490,9 @@ function manage_worker(
             worker = robust_start_worker(proj_name, nworker_threads, worker_init_expr, ntestitems)
         end
         testitem.workerid[] = worker.pid
+        # TODO: should the testitem's own timeout take precedence or should we just take the `max`?
+        # timeout = max(something(testitem.timeout, 0.0), default_timeout)
+        timeout = something(testitem.timeout, default_timeout)
```

Comment:
user should definitely take precedence. It can be useful to set a lower bound here so that for tests that should go fast but are at risk of blowup, you can set a lower timeout than the default so those timeouts can catch regressions more often